### ISSUE TITLE
[2019.12] PHP Fatal error Suddenly I get these messages today.

### ISSUE THREAD

**Issue by copiis** [friendica/friendica#8412]
```
PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Friendica\Util\Network::isUrlBlocked() must be of the type string, null given, called in /src/Protocol/ActivityPub/Transmitter.php on line 631 and defined in /src/Util/Network.php:477
Stack trace:
#0 /src/Protocol/ActivityPub/Transmitter.php(631): Friendica\Util\Network::isUrlBlocked(NULL)
#1 a/src/Worker/Notifier.php(722): Friendica\Protocol\ActivityPub\Transmitter::fetchTargetInboxes(Array, 594)
#2 /src/Worker/Notifier.php(164): Friendica\Worker\Notifier::activityPubDelivery('wall-new', Array, Array, Array, 20, '2020-03-12 10:3...', Array)
#3 [internal function]: Friendica\Worker\Notifier::execute('wall-new', '10228990')
#4 /src/Core/Worker.php(400): call_user_func_array('Friendica\\Worke...', Array)
#5 /src/Core/Worker.php(299): Friendica\Core\Worker::execFunct in /home/users/meld/www/friendica/src/Util/Network.php on line 477
```

**Comment by tobiasd** [comment]
part of #7998

**Comment by tobiasd** [comment]
see https://github.com/friendica/friendica/issues/7998#issuecomment-596073861

hopefully fixed in #8368

**Comment by MrPetovan** [comment]
Duplicate of https://github.com/friendica/friendica/issues/7998#issuecomment-596073861
